AccessCheckoutSDK/pacts/access-checkout-ios-sdk-sessions.json

```diff
@@ -0,0 +1,119 @@
+{
+    "metadata": {
+        "pactSpecification": {
+            "version": "2.0.0"
```

Comment:
The pact specification version we use is 3.0.0, can you update it in the source code and regenerate the contarcts again? (and re-upload them on the broker then)

Reply:
i found this line in the schemes:
scriptText = "${SRCROOT}/pact/bin/pact-mock-service start --pact-specification-version 2.0.0

it seems like we are using version 2.0.0? this is auto generated from what I can tell?